portway/portway
# Cleaning up org redux flow

* Adds in billing fetch / update redux actions
* Adds in UI state for open/close of stripe form
* Adds icons for known Stripe card types

### To test:

1. Use different card numbers from https://stripe.com/docs/testing#cards
2. See different icons for each card
3. Test that there is no card information at all
4. Form should disable and not be able to be submitted twice
